{"name": "CLV Custodial账户赎回数量超过质押金额", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_stake_custodial_fail.Test_stake_success object at 0x105a18e80>, test_title = 'CLV Custodial账户赎回数量超过质押金额'\nnetworkCode = 'CLV', symbol = 'CLV', type = 'un_stake', address = '5CFStFse5QY5dyfeHeTDSnMeBBRgxXse2D8k1TUbM67HeA9h'\namount = '100', status_code = 400, code = 2100000\n\n    @allure.story(\"Custodial Stake Fail!\")\n    @allure.title('{test_title}')\n    @pytest.mark.parametrize('test_title,networkCode,symbol,type,address,amount,status_code,code', test_data)\n    def test_custodial(self,test_title,networkCode,symbol,type,address,amount,status_code,code):\n    \n        with allure.step(\"浏览器查询from账户balance信息\"):\n            balance = Httpexplore.Balances_explore.query(networkCode,address,symbol)\n    \n        with allure.step(\"查询from账户holder信息\"):\n            holder = Http.HttpUtils.holders(networkCode=networkCode,symbol=symbol,address=address)\n            assert holder.status_code ==200\n>           quantity = Decimal(holder.json()[\"list\"][0][\"quantity\"])\nE           IndexError: list index out of range\n\nTestcase/Test_Cloud/Test_Stake/Test_Custodial/test_stake_custodial_fail.py:75: IndexError"}, "steps": [{"name": "浏览器查询from账户balance信息", "status": "passed", "start": 1675928860179, "stop": 1675928860455}, {"name": "查询from账户holder信息", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"/Users/lilong/Documents/Test_Api/Testcase/Test_Cloud/Test_Stake/Test_Custodial/test_stake_custodial_fail.py\", line 75, in test_custodial\n    quantity = Decimal(holder.json()[\"list\"][0][\"quantity\"])\n"}, "start": 1675928860455, "stop": 1675928860743}], "attachments": [{"name": "log", "source": "87a2ce11-521e-4fbd-94f8-c4d53673dc32-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_title", "value": "'CLV Custodial账户赎回数量超过质押金额'"}, {"name": "networkCode", "value": "'CLV'"}, {"name": "symbol", "value": "'CLV'"}, {"name": "type", "value": "'un_stake'"}, {"name": "address", "value": "'5CFStFse5QY5dyfeHeTDSnMeBBRgxXse2D8k1TUbM67HeA9h'"}, {"name": "amount", "value": "'100'"}, {"name": "status_code", "value": "400"}, {"name": "code", "value": "2100000"}], "start": 1675928860179, "stop": 1675928860744, "uuid": "bdc4a997-9ac0-4f29-93b7-d3fdd434357f", "historyId": "b57023b3c47f97fcd7391d812700db77", "testCaseId": "ac4e75e203c8633f0adfd84543a62515", "fullName": "Testcase.Test_Cloud.Test_Stake.Test_Custodial.test_stake_custodial_fail.Test_stake_success#test_custodial", "labels": [{"name": "feature", "value": "Stake!"}, {"name": "story", "value": "Custodial Stake Fail!"}, {"name": "parentSuite", "value": "Testcase.Test_Cloud.Test_Stake.Test_Custodial"}, {"name": "suite", "value": "test_stake_custodial_fail"}, {"name": "subSuite", "value": "Test_stake_success"}, {"name": "host", "value": "lilongdeMacBook-Pro.local"}, {"name": "thread", "value": "1070-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase.Test_Cloud.Test_Stake.Test_Custodial.test_stake_custodial_fail"}]}